build.gradle

```diff
@@ -189,7 +189,9 @@ project ('spring-kafka') {
 		compile "org.springframework:spring-context:$springVersion"
 		compile "org.springframework:spring-messaging:$springVersion"
 		compile "org.springframework:spring-tx:$springVersion"
-		compile "org.springframework.retry:spring-retry:$springRetryVersion"
+		compile ("org.springframework.retry:spring-retry:$springRetryVersion") {
+			exclude group: 'org.springframework'
```

Comment:
I suggest to do like this `exclude group: 'org.springframework', module: 'spring-core'` and back-port it into the `2.0.x` and `2.1.x` as well:
1. Those versions suffer the same dependency issue
2. We need to be compatible with the Spring IO Platform there.

Otherwise - LGTM!

Reply:
Hi @artembilan ,
I'll modify the way you suggest. Should I close this PR and open two new ones linked to 2.0.X and 2.1.X ?

Reply:
No, you don't need to do that - we will simply `cherry-pick` your commit to those versions.

Thanks